cnab/app/run

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+action=$CNAB_ACTION
+name=$CNAB_INSTALLATION_NAME 
+echo $NODE_PORT
+use_node_port=""
+if [ "$NODE_PORT" = true ]; then
+    use_node_port="--set gateways.istio-ingressgateway.type=NodePort"
+fi
+
+case $action in
+    install)
+        echo "Installing helm client"
+        helm init --client-only
```

Comment:
if we continue to use tiller, we'll also need to expose the common tiller connection flags. The default tiller config is not secured for production clusters as any workload in the cluster can communicate with tiller and get it to install new things using tiller's service account.